client/app/components/manager-navbar/manager-navbar-menu/manager-navbar-menu.html

```diff
@@ -0,0 +1,37 @@
+<!-- <div class="oui-navbar-menu oui-navbar-menu_fixed oui-navbar-menu_right"> -->
+    <header class="oui-navbar-menu__header">
+        <oui-back-button class="oui-navbar-menu__back"
+            on-click="$ctrl.toggleMenu($ctrl.menuName, true)">
+        </oui-back-button>
+        <span class="oui-navbar-menu__breadcrumb" data-ng-bind="::$ctrl.headerBreadcrumb"></span>
+        <h3 class="oui-navbar-menu__title" data-ng-bind="::$ctrl.headerTitle"></h3>
+    </header>
+    <ul class="oui-navbar-list">
+        <li class="oui-navbar-list__item" data-ng-repeat="menuLink in $ctrl.menuLinks track by $index">
+            <a class="oui-navbar-link"
+                data-ng-if="::!menuLink.subLinks"
+                data-ng-click="$ctrl.toggleMenu()"
```

Comment:
Another solution can be to listen the event `$stateChangeSuccess`

Reply:
For this case, yes.
But we have links with `target="_blank"`.